superset/dashboards/api.py

```diff
@@ -455,6 +453,43 @@ def get_charts(self, id_or_slug: str) -> Response:
         except DashboardNotFoundError:
             return self.response_404()
 
+    @expose("/created_by_me/", methods=["GET"])
+    @protect()
+    @safe
+    @statsd_metrics
+    @event_logger.log_this_with_context(
+        action=lambda self, *args, **kwargs: f"{self.__class__.__name__}.get",
+        log_to_statsd=False,
+    )
+    def created_by_me(self) -> Response:
+        """Gets all dashboards created by the current user
+        ---
+        get:
+          description: >-
+            Gets all dashboards created by the current user
+          responses:
+            200:
+              description: Dashboard
+              content:
+                application/json:
+                  schema:
+                    type: object
+                    properties:
+                      result:
+                        type: array
+                        items:
+                          $ref: '#/components/schemas/DashboardCreatedByMeResponseSchema'
+            401:
+              $ref: '#/components/responses/401'
+            403:
+              $ref: '#/components/responses/403'
+            404:
+              $ref: '#/components/responses/404'
+        """
+        dashboards = DashboardDAO.get_dashboards_created_by(g.user.id)
```

Comment:
One nit here - Ideally, we wouldn't access DAO methods statically as one of the benefits of DAOs is that they should improve composability of apps. i.e. you might want to implement several variations of a given DAO (one might use SQLA, whereas the other just uses SQL.exec, for example). Static references force us to find all references and update inline and don't allow for composability without import aliasing, etc.

I totally get that this is just a simple use case, and adding a command here (one more indirection layer) seems a bit overkill.

Reply:
Sort of a similar question, but would it be possible to use the existing resource name of "dashboards" and just use a query string and create a filter for this endpoint instead of a new one? Just thinking along the lines of how can we make this more restful.

Reply:
I get what your saying @craig-rueda, but this is kind of the current pattern for custom get's, also @eschutho has a very good point here, I'll refactor this and make a custom filter for the `get_list` endpoint

Reply:
I agree with @eschutho , leveraging the existing GET with a custom filter does seem like the cleanest solution.

Reply:
changed